Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -51,7 +51,7 @@ protected override void OnInitialized()
 
         if ( ParentDataGrid is not null )
         {
-            ParentDataGrid.AddColumn( this );
+            ParentDataGrid.AddColumn( this, true );

```

Comment:
Although tools like Resharper or Visual Studio  can display the argument names - GitHub cannot (yet). Does it maybe make sense to work with named arguments here?

```suggestion
            ParentDataGrid.AddColumn( this, suppressSortChangedEvent: true );

```

Reply:
I personally don't really mind it. But we don't generally make a point to do it on the code base, so that'd be a bit inconsistent.
I agree it does provide more readability though. 

I personally disregard using because any argument name change then leads anyone who was using the named argument to also having to rename.

@stsrki any thoughts?

Reply:
I've been using it in the past on different projects.

But now, it is best to be consistent with the existing codebase.

Reply:
In that case let's disregard this suggestion, but thanks for pointing it out @danm-de